layers/+spacemacs/spacemacs-editing-visual/README.org

```diff
@@ -17,3 +18,24 @@ edited line in Spacemacs.
 - Highlighting of different indentations
 - Automatic highlighting of numbers
 - Automatic highlighting of parentheses
+* Mixed pitch fonts support
```

Comment:
Please also add a warning/note about the interaction with company.
See `Interaction with company` section on its repository.